test/unit/spec/MfaVerify_spec.js

```diff
@@ -482,7 +482,7 @@ function (Okta,
           name: 'AuthApiError',
           message: message,
           statusCode: code,
-          xhr: Util.transformMockXHR(resError)
+          xhr: resError
```

Comment:
nit:
shall we follow same pattern in other tests that write out the actual error?

Reply:
Yes! My sleep deprivation caused me to skip over a few. I'll update that now. Thanks!

Reply:
Alright, I guess I wasn't too crazy. We're passing in the error object when calling this function `expectError` or `expectErrorEvent`.

Here are the usages:

- [Google TOTP](https://github.com/okta/okta-signin-widget/pull/594/files#diff-4b2ed50d042dededa6f2af06dc054199R941)
- [RSA TOTP](https://github.com/okta/okta-signin-widget/pull/594/files#diff-4b2ed50d042dededa6f2af06dc054199R1023)
- [SMS](https://github.com/okta/okta-signin-widget/pull/594/files#diff-4b2ed50d042dededa6f2af06dc054199R1441)
- [Call](https://github.com/okta/okta-signin-widget/pull/594/files#diff-4b2ed50d042dededa6f2af06dc054199R1717)
- [Email](https://github.com/okta/okta-signin-widget/pull/594/files#diff-4b2ed50d042dededa6f2af06dc054199R2018)
- [Custom SAML Factor](https://github.com/okta/okta-signin-widget/pull/594/files#diff-4b2ed50d042dededa6f2af06dc054199R3234)
- [Custom OIDC Factor](https://github.com/okta/okta-signin-widget/pull/594/files#diff-4b2ed50d042dededa6f2af06dc054199R3319)
- [Password](https://github.com/okta/okta-signin-widget/pull/594/files#diff-4b2ed50d042dededa6f2af06dc054199R3452)